app/helpers/friendships_helper.rb

```diff
@@ -0,0 +1,13 @@
+module FriendshipsHelper
+  def friend_request?(user)
```

Comment:
- [ ] There should be NO application logic in the views nor helpers:

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)

All methods with business logic, should be in models. CRUD (create, read, update, delete) is business logic.

Reply:
fixed creating business logic and partials